src/build.ts

```diff
@@ -78,7 +78,7 @@ const fetchAndUnpack = async function (platarch: Platarch): Promise<void> {
 };
 
 const tarx = async function (inputStream: Readable): Promise<void> {
-  return await new Promise((resolve, reject) => {
```

Comment:
One of the newly recommended typescript-eslint rules suggests that we should not explicitly return here since void is being returned in all cases.

Reply:
Huh, that's funny. There is defs an opposing eslint rule that says you need to be explicit about this. Not sure what my pref is to be honest.

Reply:
If the biggest change to adding 'no-confusing-void-expression' is this or the following syntax change. I strongly vote to _not_ make this change. It seems like it is making the code less clear IMO?

Reply:
I argee, the "no-confusing-void-expression" rule is new with the major version bump, and defaults to "error".  I was originally gonna just turn it off,  but I talked myself into trying it out.  I'm definitely on board to setting it to "off".  I'll update the PR